workloads/virtual-machines/unresponsive-nodes.md

```diff
@@ -0,0 +1,89 @@
+# Detecting and resolving Node issues
+
+KubeVirt has its own node daemon, called `virt-handler`. In addition to the
```

Comment:
Suggest removing first-person pronouns (our, us):

KubeVirt has its own node daemon, called `virt-handler`. In addition to the usual k8s methods of detecting issues on nodes, the `virt-handler` daemon has its own heartbeat mechanism. This allows for fine-tuned error handling of `VirtualMachineInstances`.